pingdom-checks/claims-attributes.ping

```diff
@@ -0,0 +1,29 @@
+# -*- sh -*-
+
+CLAIMS_SLACK_ID="$SLACK_VASDVP_MONITORING,$SLACK_API_ALARMS_BENEFITS"
+CLAIMS_BACKEND_SLACK_ID="$SLACK_LIGHTHOUSE_ALERTS,$SLACK_API_ALARMS_BENEFITS"
```

Comment:
unused variable `CLAIMS_BACKEND_SLACK_ID`

Reply:
Ah, you're right.  Darn copy/paste.
Removed [here](https://github.com/department-of-veterans-affairs/lighthouse-external-monitoring/pull/62/commits/117c0d5c3297837cf2b44199d34199e03fef54b9).